lib/sprockets.rb

```diff
@@ -147,7 +147,9 @@ module Sprockets
     env.paths
   end
   register_dependency_resolver 'file-digest' do |env, str|
-    env.file_digest(env.parse_file_digest_uri(str))
+    env.file_digest( # lib/sprockets/base.rb:47
```

Comment:
We have more than one `file_digest` :frowning:. This PR is really messy, needs a major-major cleanup